packages/snap-store-mobx/src/Storage/StorageStore.ts

```diff
@@ -106,6 +108,7 @@ export class StorageStore {
 			if (value && typeof value[p] != 'undefined') {
 				value = value[p];
 			} else {
+				// @ts-ignore
 				value = undefined;
```

Comment:
Should this be `value = {}` ?

Reply:
Changed to `{}` but still need to return `undefined` here